schema/2.0.0/corpora_schema.md

```diff
@@ -37,7 +37,7 @@ This document is organized by:
 
 * **AnnData** - The canonical data format for the cellxgene Data Portal is HDF5-backed [AnnData](https://anndata.readthedocs.io/en/latest) as written by version 0.7 of the anndata library.  Part of the rationale for selecting this format is to allow cellxgene to access both the data and metadata within a single file. The schema requirements and definitions for the AnnData `X`, `obs`, `var`, `obsm`, and `uns` attributes are described below.
 
-* **Organisms**. Data MUST be either Human or Mouse data. No other organisms are accepted by the cellxgene Data Portal. 
+* **Organisms**. Genes MUST be either from Human or Mouse. Multi-organism data is accepted as long as Human or Mouse orthologs are used.
```

Comment:
How do we message the addition of SARS-CoV-2?

Reply:
which begs the same question for other mouse/human infectious agents or microbiota

Perhaps a path is:
* host / primary organism must be human or mouse
* data may include mouse or human biota (is there a better term?), as long as there are no gene IDs that conflict with the pinned IDs

Reply:
> host / primary organism must be human or mouse

This PR allows **all** organisms in the NCBITaxon - but, their genes in `var` must then be orthologous genes in the pinned ontologies.

Reply:
How about?

> Genes MUST be either from Human, Mouse, or SARS-CoV-2  for Human cells from COVID-19 donors. Multi-organism data is accepted as long as Human or Mouse orthologs are used.

Reply:
_Data MUST be from an organism defined in the NCBI organismal classification. For data that is neither Human, Mouse, nor SARS-COV-2 for Human cells, orthologous genes from the pinned Human and Mouse gene annotations are REQUIRED._